src/app/my-holdings/components/backing/BackersAllocations.tsx

```diff
@@ -0,0 +1,29 @@
+import { BuilderAllocationBar } from '@/app/backing/components/BuilderAllocationBar'
+import { RightArrowIcon } from '@/components/Icons'
+import { Span } from '@/components/TypographyNew'
+import { useRouter } from 'next/navigation'
+
+export const BackersAllocations = () => {
+  const router = useRouter()
+  return (
+    <div className="flex flex-col pt-5 items-start gap-3 grow">
+      <BuilderAllocationBar
```

Comment:
We are missing the bottom legents
<img width="584" height="66" alt="Screenshot 2025-07-23 at 10 55 10 PM" src="https://github.com/user-attachments/assets/92f10b53-2c75-4126-b7f4-9c3d8ece80d7" />
.